<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerInfo</class>
 <widget class="QWidget" name="ServerInfo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>684</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWebView" name="webView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="serverDetails_groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 5px;
/*    margin-top: 1ex;*/ /* leave space at the top for the title */
 }

 QGroupBox::title {
background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
/*     padding:2 13px;*/
    padding: 3 3px;
/*     subcontrol-origin: margin;*/
 }</string>
     </property>
     <property name="title">
      <string>Server Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
